internal/pnp/manifestparser.go

```diff
@@ -0,0 +1,315 @@
+package pnp
```

Comment:
We should add a comment explaining what is PnP along with a link to the specification.